src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -152,6 +152,8 @@ public class SnowflakeSinkConnectorConfig {
           + "By default messages are not sent to the dead letter queue. "
           + "Requires property `errors.tolerance=all`.";
 
+  public static final String SCHEMATIZATION_ENABLE_CONFIG = "schematization.enable";
```

Comment:
I would keep this as `enable.schematization`

Reply:
Done.